doc/src/docutils/tools/doxygen_xml2qbk/doxygen_xml_parser.hpp

```diff
@@ -390,23 +391,6 @@ static void parse_element(rapidxml::xml_node<>* node, configuration const& confi
             el.location = loc;
             el.line = atol(get_attribute(node, "line").c_str());
         }
-        else if (full == ".detaileddescription.para.qbk")
-        {
-            el.qbk_markup.push_back(markup(node->value()));
-        }
-        else if (full == ".detaileddescription.para.qbk.after.synopsis")
-        {
-            el.qbk_markup.push_back(markup(markup_after, markup_synopsis, node->value()));
-        }
-        else if (full == ".detaileddescription.para.qbk.before.synopsis")
-        {
-            el.qbk_markup.push_back(markup(markup_before, markup_synopsis, node->value()));
-        }
-        else if (full == ".detaileddescription.para.qbk.distinguish")
-        {
-            el.additional_description = node->value();
-            boost::trim(el.additional_description);
```

Comment:
This block is moved below. The actual bug was that some `.qbk` were missed because too explicit full paths.
In the past the XML was apparently a bit different, because it did work before.